app/console.go

```diff
@@ -35,30 +37,36 @@ func RunConsole(ps *peer.PeerStore) {
 	shell.AddCmd(&ishell.Cmd{
 		Name: "address",
 		Help: "show the address this host is listening on",
-		Func: listenAddr,
+		Func: func(ctx *ishell.Context) {
+			listenAddr(ctx, peerStore)
+		},
 	})
 	shell.AddCmd(&ishell.Cmd{
 		Name: "peers",
 		Help: "show the peers this host is connected to",
-		Func: peers,
+		Func: func(ctx *ishell.Context) {
```

Comment:
Why wrap function calls in new anonymous functions? Couldn't you, for example, make `peers` take `ctx` and then just do the stuff `peers` currently does? That would make the code shorter and more readable.